docker/serverless/gcloud_build_image

```diff
@@ -92,10 +92,12 @@ if [ -z "${ESP_FULL_VERSION}" ]; then
 fi
 echo "Building image for ESP version: ${ESP_FULL_VERSION}"
 
-cd "$(mktemp -d /tmp/docker.XXXX)"
+tempdir="$(mktemp -d /tmp/docker.XXXX)"
+cd "${tempdir}"
 
 # Be careful about exposing the access token.
-curl --fail -o "service.json" -H "Authorization: Bearer $(gcloud auth print-access-token)" \
+access_token="$(gcloud auth print-access-token)"
```

Comment:
Is this token not exposed in this case? Can we add --scopes=...?

Reply:
Yeah, this doesn't solve the problem at all because `${access_token}` is still in the command line below. It needs to be in standard input or somewhere else that's private. For standard input:
```sh
curl ... -H @- ... <<EOF
Authorization: Bearer ${access_token}
EOF
```

Reply:
Done